packages/grafana-prometheus/src/querybuilder/components/MetricsLabelsSection.tsx

```diff
@@ -254,3 +254,11 @@ async function getMetrics(
     description: getMetadataString(m, datasource.languageProvider.metricsMetadata!),
   }));
 }
+
+export function getMetadataString(metric: string, metadata: PromMetricsMetadata): string | undefined {
+  if (!metadata[metric]) {
+    return undefined;
```

Comment:
nit: maybe just return?
```suggestion
    return;
```